lib/src/base-styles/_nx-pagination.scss

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-btn-bar--pagination {
+  text-align: center;
+
+  .nx-btn {
+    height: 40px;
+    min-width: 40px;
+    margin: 0 $nx-spacing-xs 0 0;
+    padding: 0 $nx-spacing-md;
```

Comment:
The buttons are ending up `42px` wide rather than `40px` which I wasn't seeing before but now I am, I'm thinking that we should subtract the borders from the padding as we have done elsewhere.

Reply:
> The buttons are ending up 42px wide rather than 40px which I wasn't seeing before but now I am, I'm thinking that we should subtract the borders from the padding as we have done elsewhere.

See https://sonatype.slack.com/archives/CJUF9G7T2/p1604064001371900 .  The 40px is a minimum which doesn't really have any effect anywhere now that I've made the arrows slightly larger than they were.